### PR TITLE
Added virus assignment

### PR DESCRIPTION
**Pull request by janettelau**

### CODE REVIEW

File: virus/Blumevirus_macaceinfi.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+### A Close Relative of Leviviridae sp.
+Based on results generated using the blastx tool in NCBI, our novel virus is most closely related to the virus Leviviridae sp., with a percent identity of 44.54% to its RdRP protein based off a Third Party Annotation (TPA), and a percent identity of 42.66% to the whole RdRP protein. Leviviridae sp. also belongs to the family Blumeviridae, however, because B. macaceinfi and Leviviridae sp. are only related on the class level, it is not extremely significant in terms better understanding our novel virus. More details confirming the structure and particular features of B. macaceinfi are required to further categorize it into the most suitable genus.
+
+### Virus Genome
+
+![Genome Map](img/genomemap.png)
```
Comment (ababaian) [inline virus/Blumevirus_macaceinfi.md]: Can you use a `<virus_name>` folder path for the images/data files please. Look at how the data is organized in the repository currently.

File: virus/img/pymol.png
Comment (ababaian) [inline virus/img/pymol.png:1]: Are these structures aligned to one another with the `super` command in pymol or are they defaults how they were loaded (in which case they are not aligned to one another).

File: virus/img/genomemap.png
Comment (ababaian) [inline virus/img/genomemap.png:1]: change path to `virus_name` folder for images please.

**Review by ababaian — CHANGES_REQUESTED** [review]

File: virus/img/pymol.png
Comment (janettelau) [reply virus/img/pymol.png:1]: They are defaults, not aligned.

File: virus/Blumevirus_macaceinfi.md (hunk 1 above)
Comment (janettelau) [reply virus/Blumevirus_macaceinfi.md]: Just added. Thanks.

**Review by ababaian — APPROVED** [review]
Nice well done.

**Pull request merged**